conanfile.py

```diff
@@ -0,0 +1,8 @@
+from conan import ConanFile
+
+class CompressorRecipe(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeToolchain", "CMakeDeps"
+
+    def build_requirements(self):
```

Comment:
you are missing `build()`

https://docs.conan.io/2.10/reference/conanfile/methods/build.html#build

this way you can build it using only conan

with `conan install `and `conan build` commands